.github/workflows/move-inprogress-to-review.yml

```diff
@@ -0,0 +1,19 @@
+name: Move from In Progress to Review
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  add-comment:
+    if: github.event.label.name == 'ready-for-review' # Verifier le nom du label
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f # Verifier l'action
```

Comment:
@adrien5054 thank you so much for the PR! 

I believe this doesn't work, we need to add the version of the action, additionally this action doesn't have `project` and `column` parameters.

Could you please have a look :)

Reply:
Also, we don't need the `test.txt` file, we can remove it :)